corehq/apps/export/templates/export/spec/ko/mocha.html

```diff
@@ -10,6 +10,5 @@
 {% block mocha_tests %}
 <script src="{% static 'export/spec/data/export_instances.js' %}"></script>
 <script src="{% static 'export/spec/ExportInstance.spec.js' %}"></script>
-<script src="{% static 'export/spec/ExportColumn.spec.js' %}"></script>
```

Comment:
@benrudolph Interestingly this line hasn't been causing the build to fail on travis, although it does fail locally for me, and it seems to have been there since March: https://github.com/dimagi/commcare-hq/commit/f5bd9ce321a443c89d154cb4484f6aac149d38d3#diff-8202a0df276ac248edf0135b58c2f931

I wonder if its worth investigating why it wasn't failing on Travis?